stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -338,6 +361,22 @@ public void onChildRemoved(Object parentElement, Object childElement) {
     public RGBAColor contentColor;
   }
 
+  private static class InspectNodeRequestedEvent {
```

Comment:
nit: I think we use `*Params` for events and `*Request/Response` for commands.  See `Network` for examples.